mlir/utils/jenkins/Jenkinsfile

```diff
@@ -297,6 +313,11 @@ pipeline {
                      description: 'Run weekly-only jobs')
         string(name: 'MIOpenBranch', defaultValue: 'develop',
                description: 'The MIOpen branch to be used with the job')
+        // MIGRAPHX_COMMIT is defined in the Dockerfile
+        // So in order to update the commit, please change the Dockerfile
```

Comment:
I'm still going to insist that this should be develop

Reply:
I'd defer that to the image where dependencies are installed -- I insist we need to agree on what the image uses -- and lets not decide that here.

Reply:
I don't think the image and the CI being out of sync a bit on dependency versions is fine - we've been doing it for MIOpen with no issues